flink-python/pyflink/fn_execution/beam/beam_coders.py

```diff
@@ -194,7 +194,7 @@ def _to_data_type(field_type):
                                                field_type.nullable)
             elif field_type.type_name == flink_fn_execution_pb2.Schema.TIMESTAMP:
                 return TimestampType(field_type.timestamp_info.precision, field_type.nullable)
-            elif field_type.type_name == flink_fn_execution_pb2.Schema.ARRAY:
```

Comment:
should we also add PRIMITIVE_ARRAY?

Reply:
In SQL/ Table API, the array type is the BasicArrayType, PrimitiveArrayType is not considered.